### ISSUE TITLE
Idea: Double output of Crystallization

### ISSUE THREAD

**Issue by KiwiHawk** [KiwiHawk/SeaBlock#338]
Multiply the output of MW and sludge crystallisation by a multiplier, maybe 1.5x or 2x.

This might be a bit controversial, but I really liked the idea of doubling the ore output from mineral water and sludge. It makes it a lot easier to hit the point where resources for building aren't an issue, and late game it means a smaller percentage of the base is sludge stacks, which don't even get any meaningful upgrades past green science besides higher tier buildings and modules.

I personally think that making the early game resource crunch much less of a thing through increasing production would go a long way towards making the modpack feel less grindy at the start, since every other modpack including Py allows you to get free resources at large rates from the ground and having to struggle to get a trickle of plates for expensive buildings makes Sea Block feel uniquely grindy at the start.

I did a bit of maths on the doubling crystallisation recipe output idea, and got the following estimated values:

For 3 T4 electrolysers running fast dirt water electrolysis, with T3 modules, all outputs redirected to mineral water, and T1 mixed ore sorting used (i.e. the most potential for a loop):
Base output (No mixed ore sorting, no prod mods): 90 crushed ore from slag + ~29 crushed ore from mineral water crystallisation
Max output (T1 mixed ore sorting with recycled slag, T3 prod mods): 120.4 crushed ore from slag + ~31 crushed ore from mineral water crystallisation
% increase: ~1.25x

### Recycling Proportion:
Mineral Water: Crushed Stone from Crushed ore is ~23.8% of the mineral water from all other sources
### Slag:
Slag from T1 mixed ore sorting is ~32.4% of the original slag produced from dirt water electrolysis -> Leads to ~1.42x multiplier overall

If the output from the 2 crystalliser recipes is doubled:
- Crushed stone from Crushed ore is doubled to be ~47.6% of the mineral water from all other sources -> leads to about 1.2x increase in mineral water crystallisation after doubling (2.4x)
- Slag from T1 mixed ore sorting is now ~65% of the original slag produced from dirt water electrolysis -> leads to ~2.86x multiplier overall (4x)

Overall: With most optimal recipes, a 2x buff to the crystallisation recipes will lead to a ~4x buff in the most optimal case of T3 prod mods + T1 mixed ore sorting, and it's almost entirely caused by the T1 mixed ore sorting output since mineral water doesn't have a big impact.

An easy way to account for this is to nerf the slag output from the T1 mixed ore sorting, which will cause the buff to slag from mixed ore sorting to be reduced to pre-buff levels and thus lead to only a ~2.2-2.4x buff with prod 3s and mixed ore sorting

So basically, no positive feedback loops from doubling mineral water crystallisation and mineral slurry crystallisation recipes to have double output, but a significant buff to T1 mixed ore sorting that can be mitigated by halving the slag output from the mixed ore sorting recipes 

Or just keep it to incentivise mixed ore sorting, it sucks enough already anyway

**Comment by KiwiHawk** [comment]
Paraphrased from posts on Discord by BinaryPulsar.
